great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py

```diff
@@ -12,7 +10,9 @@
     InvalidExpectationConfigurationError,
     render_evaluation_parameter_string,
 )
-from great_expectations.expectations.metrics.util import parse_value_set
+
+# NOTE: Necessary to import the metrics code BEFORE expectations are run to trigger global registry
+from great_expectations.expectations.metrics.util import parse_value_set  # noqa: F401
```

Comment:
Recorded a Loom around this in the OSS channel - I think I have a solution in mind but want to field it by Zelda first